docs/source/happy.rst

```diff
@@ -300,6 +300,25 @@ Or exclude matching tests from being run:
    $ bin/test -t \!robot
 
 
+Running tests with the Google Chrome browser
+--------------------------------------------
+
+In our setup, robot framework uses Firefox to run the robot tests.
```

Comment:
Is that true? I tried to temporarily set Chrome instead of Firefox as default browser, but still Firefox was started. That brought me to the conclusion, that Firefox might be set in config.